amd/src/mod_hsuforum_loader.js

```diff
@@ -45,6 +45,7 @@ define(['jquery'], function ($) {
 
         if (posts) {
             $(posts).each(function(){
+                removeAttagging();
```

Comment:
Might be worth moving this outside of the loop, as the selector in the function will target all nodes that meet the check first time it runs, will jsut save some time so it doesn't need to crawl the DOM for every single post

Reply:
@BrendonPretorius updated above